pandas/core/frame.py

```diff
@@ -3547,6 +3547,54 @@ def to_xml(
 
         return xml_formatter.write_output()
 
+    def to_iceberg(
+        self,
+        table_identifier: str,
+        catalog_name: str | None = None,
+        *,
+        catalog_properties: dict[str, Any] | None = None,
+        location: str | None = None,
+        snapshot_properties: dict[str, str] | None = None,
+    ) -> None:
+        """
+        Write a DataFrame to an Apache Iceberg table.
+
+        .. versionadded:: 3.0.0
```

Comment:
Could you add an experimental tag to this API as well like we did with `read_iceberg`?

Reply:
Absolutely, I forgot about that. Added it now. I also expanded the user guide docs of iceberg with `to_iceberg`, which I also had forgotten. Thanks for the review and the feedback!